CMakeLists.txt

```diff
@@ -10,14 +10,14 @@
 #     http://creativecommons.org/publicdomain/zero/1.0/
 #
 ########################################################################
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.10.2)
```

Comment:
For this commit too, it would be nice to have some context about _why_ we require this newer version.

Reply:
Yes, it was discussed above. But that never made it into the commit message.

Reply:
I would propose the following commit message:
```
Bump the required cmake version to `3.10.2`

CMake `3.9` introduced the library target `OpenMP::OpenMP_C` which is way more portable than the old compiler flag based approach. Due to the required version bump we surveyed the available CMake versions on widely used OS distributions. With said survey data (see below) we chose `3.10.2` as the new required version which only excludes RHEL7.

https://lists.llvm.org/pipermail/llvm-dev/2020-April/140578.html
```

Reply:
That's a great commit body, thanks.